app/controllers/admin/pages_controller.rb

```diff
@@ -95,31 +101,5 @@ def prepopulate_new_form(slug)
                 Page.new
               end
     end
-
-    def update_and_overwrite_landing_page
```

Comment:
we don't need the whole concept of "overwrite landing page" anymore because the `Page` model will take care of this automatically

Reply:
It's so nice to see this gone!